### PR TITLE
Add ddns-domainname and ddns-rev-domainname option

### PR DESCRIPTION
**Pull request by xcompass**
ddns-domainname is needed when multiple domains set for a zone
ddns-rev-domainname is needed to support reverse lookup update

### CODE REVIEW

**Comment by mmoll** [comment]
While I'm not against this change as such, I'd like to ask if using the generic options paramater added in #49 would solve this particular problem for you.

**Comment by xcompass** [comment]
I don't think those ddns settings can be set through option, as it prepend the "option" in front of the settings. e.g.:
```
option ddns-domainname xxx.xxx.xx.xx
```

Though the correct setting should be:
```
ddns-domainname xxx.xxx.xx.xx
```

Unless I missed something.

**Comment by mmoll** [comment]
I see, let's go for it, then :)
A spec test would be nice...

File: templates/dhcpd.conf.erb [hunk 1]
```diff
@@ -23,6 +23,13 @@ ddns-update-style interim;
 update-static-leases on;
 use-host-decl-names on;
 
+<% if @ddns_domainname -%>
```
Comment (ekohl) [inline templates/dhcpd.conf.erb]: I'm worried about empty string behaviour here. Maybe just to be sure ```if @ddns_domainname and !@ddns_domainname.empty?```

**Comment by xcompass** [comment]
@mmoll tests added and updated based on @ekohl 's comment

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, thanks @xcompass!

**Pull request merged**